### PR TITLE
Update model SQLs for Teradata

### PR DESCRIPTION
**Pull request by adamtwo**
## What are you changing in this pull request and why?

The original Teradata quickstart that we contributed before the launch had several discrepancies. This PR reflects the early feedback from the people who tried to complete the quickstart.

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [x] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/guides/teradata-qs.md [hunk 1]
```diff
@@ -59,50 +59,48 @@ If you created your Teradata Vantage database instance at https://clearscape.ter
 
 :::
 
-1. Use your preferred SQL IDE editor to create two databases: `jaffle_shop` and `stripe`:
+1. Use your preferred SQL IDE editor to create database `jaffle_shop`:
```
Comment (mirnawong1) [inline website/docs/guides/teradata-qs.md]:
```suggestion
1. Use your preferred SQL IDE editor to create the database, `jaffle_shop`:
```

File: website/docs/guides/teradata-qs.md [hunk 2]
```diff
@@ -59,50 +59,48 @@ If you created your Teradata Vantage database instance at https://clearscape.ter
...
+        paymentmethod varchar (100),
+        amount integer
+    )
+    USING (
+        LOCATION ('/gs/storage.googleapis.com/clearscape_analytics_demo_data/dbt/raw_payments.csv')
+    )
+    NO PRIMARY INDEX;
+    ```
 
 ## Connect dbt cloud to Teradata
```
Comment (mirnawong1) [inline website/docs/guides/teradata-qs.md]:
```suggestion
## Connect dbt Cloud to Teradata
```

**Review by mirnawong1 — APPROVED** [review]
hey @adamtwo thanks for openign this up! I've approved it but have a couple of small suggestions. i can't make those changes myself - can you make those tweaks on your end?

**Comment by adamtwo** [comment]
> hey @adamtwo thanks for openign this up! I've approved it but have a couple of small suggestions. i can't make those changes myself - can you make those tweaks on your end?

Thanks for the suggestions, I've accepted them. I'm good to merge the PR.

**Review by mirnawong1 — APPROVED** [review]

**Pull request merged**